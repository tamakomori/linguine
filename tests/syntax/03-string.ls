func main() {
    a = "aaaaa";
    b = 123;
    c = 1.23;
    print("a = " + a + " len=" + length(a));
    print("a + 123 = " + (a + 123));
    print("b = " + b);
    print("c = " + c);
}
