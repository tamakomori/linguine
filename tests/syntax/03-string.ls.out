a = aaaaa len=5
a + 123 = aaaaa123
b = 123
c = 1.230000
